src/scss/main.scss

```diff
@@ -6,14 +6,11 @@
 @import 'icon-fonts/index';
 
 html {
-  line-height: 1.15;
+  @apply text-base;
 }
 
 body {
-  line-height: 1;
-
   // TODO Move to some other place so that it's more elegant.
-  font-size: 14px;
   --va-checkbox-font-size: 0.875rem;
   --va-card-box-shadow: none; // TODO Remove after https://github.com/epicmaxco/vuestic-ui/issues/3964
   --va-card-padding: 1rem;
```

Comment:
why do we have --va-button-font-size: 0.875rem; here? it sets the font size to 12.25px instead of the 14px, Considering that 14px is the default and does not require overwriting.